Detectors/TRD/base/include/TRDBase/ArrayADC.h

```diff
@@ -0,0 +1,85 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_ARRAYADC_H
+#define O2_ARRAYADC_H
+
+namespace o2
+{
+namespace trd
+{
+
+///////////////////////////////////////////////
+//                                           //
+// Container class for ADC values            //
+//                                           //
+///////////////////////////////////////////////
+
+// ported from cblume original code in AliRoot, possibly some modifications
```

Comment:
```suggestion
// reimplementation of original code by cblume in AliRoot
// uses STL vectors
```

Reply:
A bit more information about the memory organisation would be very helpful.

Reply:
Yes, the "possibly some modifications" was not the best formulation. It was intended to be edited later, once all the modifications were complete.